presto-jdbc/src/main/java/com/facebook/presto/jdbc/ColumnInfo.java

```diff
@@ -227,6 +233,8 @@ private static int getType(TypeSignature type)
                 return Types.DATE;
             case "decimal":
                 return Types.DECIMAL;
+            case "float":
+                return Types.REAL;
```

Comment:
Postgres actually implements bare `float` as `double precision`: "float with no precision specified is taken to mean double precision.". https://www.postgresql.org/docs/9.5/static/datatype-numeric.html#DATATYPE-FLOAT